packages/jest-environment-puppeteer/package.json

```diff
@@ -21,7 +21,8 @@
     "prepublishOnly": "yarn build"
   },
   "peerDependencies": {
-    "puppeteer": "^1.5.0"
+    "puppeteer": "^1.5.0",
+    "puppeteer-firefox": "^0.5.0"
```

Comment:
We will need to remove peer dependencies, else it will emits a warning.

Reply:
Hmm, this one I don't follow. Peer dependency is needed for puppeteer-firefox the same way it's needed for puppeteer. Can you please elaborate?

Reply:
Peer dependencies display warning if not installed. In this case we will not only one of the two peer dependencies. It means we will have a warning in the console. The only solution is to completely remove peer dependencies.

Reply:
>  The only solution is to completely remove peer dependencies.

@neoziro Done. It's still nice to have them defined, so opted for optional dependencies instead.

Reply:
`optionalDependencies` is even worse, the two dependencies will be installed. Please remove them completely.

Reply:
Done.